pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -487,8 +501,30 @@ static void cleanup_tcp_conn(struct pt_regs* ctx, conn_tuple_t* tup) {
         conn.conn_stats = *cst;
     }
 
-    // Send the connection data to the perf buffer
-    bpf_perf_event_output(ctx, &tcp_close_event, cpu, &conn, sizeof(conn));
+    // Batch TCP closed connections before generating a perf event
+    batch_t zero_batch = {};
+    bpf_map_update_elem(&tcp_close_batch, &cpu, &zero_batch, BPF_NOEXIST);
+
+    batch_t *batch_ptr = bpf_map_lookup_elem(&tcp_close_batch, &cpu);
+    if (batch_ptr == NULL) {
+        return;
```

Comment:
can we add a comment explaining that this should never happen since we just did an update_elem ?